### DISCUSSION TITLE
Sugestão a respeito de agradar comunidade mobile

### DISCUSSION THREAD

**Discussion by Batatoones** [leogianfagna/IssueTracker#955] [Opinioes]
### Servidor afetado

Todos

### Descrição

Como possui uma parcela de mobiles que não consegue receber as recompensas de votos dentro do servidor, o intuito dessa sugestão é agradar todos.
Como comentado, criar uma lista de tarefas diárias que recompensaria os jogadores ao realizarem da mesma maneira que ocorre para os votos. Fazer isso de maneira que seja dedicada aos mobiles apenas tendo em vista que eles enfrentam os problemas.

Contras: Iria ter menos votos dos mobiles mas ainda sim teria players dentro do servidor e a gente sabe que número de jogadores atraí mais jogadores.

**Comment by leogianfagna** [comment]
isso foi corrigido
